translate/tests/system.py

```diff
@@ -54,19 +54,17 @@ def test_detect_language(self):
         self.assertEqual(detections[2]["language"], "fr")
 
     def test_translate(self):
-        values = ["hvala ti", "dankon", "Me llamo Jeff", "My name is Jeff"]
+        values = ["petnaest", "dek kvin", "Me llamo Jeff", "My name is Jeff"]
         translations = Config.CLIENT.translate(
             values, target_language="de", model="nmt"
         )
         self.assertEqual(len(values), len(translations))
 
         self.assertEqual(translations[0]["detectedSourceLanguage"].lower(), "hr")
-        self.assertEqual(translations[0]["translatedText"].lower(), "danke")
+        self.assertEqual(translations[0]["translatedText"].lower(), u"f\xfcnfzehn")
```

Comment:
This example would be clearer with an actual u-umlaut character, i.e., `u"fünfzehn"` (we use the sharp-S below, for instance).